deltachat-ffi/deltachat.h

```diff
@@ -5303,16 +5291,22 @@ int             dc_contact_is_key_contact    (dc_contact_t* contact);
 /**
  * Return the contact ID that verified a contact.
  *
- * If the function returns non-zero result,
- * display green checkmark in the profile and "Introduced by ..." line
- * with the name and address of the contact
- * formatted by dc_contact_get_name_n_addr.
+ * Having a verfier does not necessarily mean that
+ * you can add the contact to protected chats.
+ * For that, check dc_contact_is_verified()
+ * and display the information as follows:
+ *
+ * - If dc_contact_get_verifier_id() != 0,
+ *   display text "Introduced by ..."
+ *   with the name and address of the contact
+ *   formatted by dc_contact_get_name_n_addr().
+ *   Prefix the text by a green checkmark if dc_contact_is_verified() != 0.
```

Comment:
Same here: If there is a verifier, then this means the contact is verified, i.e. can be added to protected chats. So, at least today, it wouldn't be necessary to make the UIs handle this case.